server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -666,7 +666,7 @@ int main(int argc, char** argv)
 
     const auto thread_count = [] {
         try {
-            return irods::get_advanced_setting<const int>(irods::CFG_MAX_NUMBER_OF_CONCURRENT_RE_PROCS);
+            return irods::get_advanced_setting<const int>(irods::CFG_NUMBER_OF_CONCURRENT_DELAY_RULE_EXECUTORS);
```

Comment:
The names of the constants for the default values returned on 643 and 675 still refer to the old names.

Reply:
Will change.

Reply:
done